src/blocks/CaseStudyParallax/index.tsx

```diff
@@ -0,0 +1,307 @@
+'use client'
+import * as React from 'react'
+
+import { BackgroundGrid } from '@components/BackgroundGrid'
+import { BackgroundScanline } from '@components/BackgroundScanline'
+import { BlockSpacing } from '@components/BlockSpacing'
+import { Button } from '@components/Button'
+import { Gutter } from '@components/Gutter'
+import { Media } from '@components/Media'
+import { QuoteIconAlt } from '@root/icons/QuoteIconAlt'
+import { Page } from '@root/payload-types'
+import { useResize } from '@root/utilities/use-resize'
+
+import classes from './index.module.scss'
+
+type Props = Extract<Page['layout'][0], { blockType: 'caseStudyParallax' }>
+
+type StickyBlockProps = Props & {
+  currentIndex: number
+}
+
+type QuoteProps = {
+  card: any
+  isVisible?: boolean
+  className?: string
+}
+
+export const QuoteBlock: React.FC<QuoteProps> = props => {
+  const { isVisible, card, className } = props
+  return (
+    <div
+      className={[isVisible && classes.isVisible, 'cols-8 grid', className]
+        .filter(Boolean)
+        .join(' ')}
+      aria-hidden={!isVisible}
+    >
+      <QuoteIconAlt className={classes.quoteIcon} />
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.quote, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        {card.quote}”
+      </div>
+
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.authorWrapper, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        <div className={classes.media}>
+          {typeof card.logo !== 'string' && <Media resource={card.logo} />}
```

Comment:
@paulpopus It doesn't look like you are handling `dark` vs `light` mode here for the logo image - if I import a `white` logo, it stays white in `light` mode